neoforge-1.21.1/src/main/resources/META-INF/neoforge.mods.toml

```diff
@@ -0,0 +1,26 @@
+modLoader="javafml"
+loaderVersion="[4,)"
+license="LGPL-3.0-or-later"
+issueTrackerURL="https://github.com/quat1024/apathy/issues"
+
+[[mods]]
+modId="apathy"
+version="${version}"
+displayName="Apathy"
+displayURL="https://github.com/quat1024/apathy/"
+logoFile="apathy-icon-128.png"
+credits="whizball1 for Apathetic Mobs, jaredl0008 and Darkhax for MultiLoader-Template"
```

Comment:
You  can add yourself to the credits if you want

Reply:
I forgot to do that on NeoForge, whoops. Did it on Fabric tho